src/test/run-pass/generator/static_generators.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
```

Comment:
Could you also add an rpass test for the "vexpr lifetime" case (aka #44197)?

Reply:
I also need to think of more tests to add here.

Reply:
I added a test case for #44197.